eng/packages/http-client-csharp/generator/TestProjects/Local/Basic-TypeSpec/src/Generated/BasicTypeSpecClient.RestClient.cs

```diff
@@ -32,7 +32,7 @@ internal HttpMessage CreateSayHiRequest(string headParameter, string queryParame
             HttpMessage message = Pipeline.CreateMessage(context, PipelineMessageClassifier200);
             Request request = message.Request;
             request.Method = RequestMethod.Get;
-            ClientUriBuilder uri = new ClientUriBuilder();
+            Azure.Core.RawRequestUriBuilder uri = new Azure.Core.RawRequestUriBuilder();
```

Comment:
I think we need to change something to add those shared code into the generated code project to let roslyn know to trim these namespaces off.

Reply:
Feel free to create an issue for this because this might not be possible right now

Reply:
https://github.com/microsoft/typespec/issues/5104 is tracking this.